### DISCUSSION TITLE
Welcome to autogpt-gui Discussions!

### DISCUSSION THREAD

**Discussion by thecookingsenpai** [thecookingsenpai/autogpt-gui#4] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by IlPiz** [comment]
Recently I tried to get Auto GPT to develop the app itself, without succeeding... in a very simple way, I asked him to insert Auto GPT from the github link inside a graphical interface, but nothing worked.
Any idea how to proceed?
